lib/govuk_publishing_components/presenters/button_helper.rb

```diff
@@ -28,14 +29,22 @@ def link?
       def html_options
         options = { class: css_classes }
         options[:role] = "button" if link?
-        options[:type] = "submit" unless link?
+        options[:type] = button_type
         options[:rel] = rel if rel
         options[:data] = data_attributes if data_attributes
         options[:title] = title if title
         options[:target] = target if target
         options
       end
 
+      def button_type
```

Comment:
Could this be simplified to something like:

```
type || "submit" unless link?

```

Reply:
Ooh that's better, thanks.

Reply:
@vanitabarrett have made those changes.